Peer/Peer.csproj

```diff
@@ -3,7 +3,6 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>net6.0</TargetFramework>
-    <ImplicitUsings>enable</ImplicitUsings>
```

Comment:
Not sure how I'd feel about having this one enabled; would take some getting used to for sure.